### PR TITLE
Set up continous integration

### PR DESCRIPTION
**Pull request by basilcea**
# Description
 Use travisCI  for continous integration
- Create travis.yml
- set up travis database
- Add coveralls
- Run travis test

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works AND the tests pass

## Link to Trello board
[Set up continous integration](https://trello.com/c/cRmsOWR5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # wtc-backend
-
-[![Coverage Status](https://coveralls.io/repos/github/where-to-code/wtc-backend/badge.svg?branch=master)](https://coveralls.io/github/where-to-code/wtc-backend?branch=master)
-
+[![Coverage Status](https://coveralls.io/repos/github/where-to-code/wtc-backend/badge.svg?branch=chore/set-up-C.I)](https://coveralls.io/github/where-to-code/wtc-backend?branch=chore/set-up-C.I)
 🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your portfolio. This is intended to be a guideline. Feel free to add your own flare to it.
```
Comment (Oyekunle-Mark) [inline README.md]: Add a free line in between the badge and the readme body.
It is preferred if the badges are at the very top of the readme, separate with a free line and let the readme title come next

File: api/server.spec.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ describe('/ [GET]', () => {
     request(server)
       .get('/')
       .expect('Content-Type', /json/)
-      .expect(200);
-      done
+      .then(res =>{
```
Comment (Oyekunle-Mark) [inline api/server.spec.js]: Add a space between the `=>` and the `{`. 
Configure and use prettier to ease the process.

File: api/server.spec.js [hunk 3]
```diff
@@ -18,14 +19,16 @@ describe('/ [GET]', () => {
         expect(res.body.message).toEqual(
           'The Where-to-Code Server is up!',
         );
+        done()
       }));
 
-  it('catches and handles bad request', () =>
+  it('catches and handles bad request', (done) =>
```
Comment (Oyekunle-Mark) [inline api/server.spec.js]: For a single argument, you can remove the parenthesis around the argument.

File: api/server.spec.js [hunk 4]
```diff
@@ -18,14 +19,16 @@ describe('/ [GET]', () => {
...
     request(server)
       .get('/very-inappropriate-url')
       .expect('Content-Type', /json/)
       .expect(404)
       .then(res => {
         expect(res.body.message).toEqual('No endpoint matches that URL.');
+        done()
```
Comment (Oyekunle-Mark) [inline api/server.spec.js]: Add a semi-colon.
Adhere to eslint rules.

File: README.md (hunk 1 above)
Comment (basilcea) [reply README.md]: Thank Oye done

File: api/server.spec.js (hunk 2 above)
Comment (basilcea) [reply api/server.spec.js]: Done. Prettier has been used

1.  prettier extensions settings enabled
2. prettier formatted

File: api/server.spec.js (hunk 3 above)
Comment (basilcea) [reply api/server.spec.js]: Done. Prettier has fixed it on format

File: api/server.spec.js (hunk 4 above)
Comment (basilcea) [reply api/server.spec.js]: Done. Semicolon and configured eslint

**Pull request merged**